src/core/types/nfts.ts

```diff
@@ -57,6 +59,7 @@ export enum SimpleHashChain {
   ProofOfPlayBoss = 'proof-of-play-boss',
   Rari = 'rari',
   RariTestnet = 'rari-testnet',
+  Sanko = 'sanko',
```

Comment:
```suggestion
  Sanko = 'sanko',
  SankoTestnet = 'sanko-testnet',
```
need this? some places are missing sanko testnet. not sure if intentional or not.